core/src/main/java/com/microsoft/alm/auth/oauth/DeviceFlowImpl.java

```diff
@@ -132,6 +138,7 @@ else if (OAuthParameter.ERROR_SLOW_DOWN.equals(errorCode)) {
                 throw new Error(e);
             }
         }
+        while (Calendar.getInstance().compareTo(expiresAt) <= 0);
```

Comment:
Regarding `intervalMilliseconds *= 2;`, the sleep time between requests increases pretty dramatically.  Can we increase a little slower, say `* 1.5`?